controllers/storagecluster/cephfilesystem.go

```diff
@@ -164,22 +164,32 @@ func (obj *ocsCephFilesystems) ensureCreated(r *StorageClusterReconciler, instan
 }
 
 func (r *StorageClusterReconciler) createDefaultSubvolumeGroup(filesystemName, filesystemNamespace string, ownerReferences []metav1.OwnerReference) error {
-	// TODO: After fix of rook issue https://github.com/rook/rook/issues/13220 add svg name spec
 
 	existingsvg := &cephv1.CephFilesystemSubVolumeGroup{}
-	err := r.Client.Get(r.ctx, types.NamespacedName{Name: defaultSubvolumeGroupName, Namespace: filesystemNamespace}, existingsvg)
+	svgName := generateNameForCephSubvolumeGroup(filesystemName)
+	err := r.Client.Get(r.ctx, types.NamespacedName{Name: svgName, Namespace: filesystemNamespace}, existingsvg)
```

Comment:
have you tested the changes locally do you see the reconcile of subvolumegroup in rook operator

Reply:
Yes